nbstripout/_nbstripout.py

```diff
@@ -428,6 +442,11 @@ def main():
     args = parser.parse_args()
     git_config = ['git', 'config']
 
+    if args.verify:
+        if not args.dry_run:
+            print("Running in verify mode, setting --dry-run")
+            args.dry_run = True
```

Comment:
```suggestion
    if args.verify and not args.dry_run:
        args.dry_run = True
```

The docstring already clarifies that `--verify` implies `--dry-run`, so I don't think the `print` here is needed?